include/boost/numeric/odeint/stepper/bulirsch_stoer_dense_out.hpp

```diff
@@ -129,6 +130,7 @@ class bulirsch_stoer_dense_out {
             else
                 m_cost[i] = m_cost[i-1] + m_interval_sequence[i];
             m_coeff[i].resize(i);
+            m_facmin_table[i] = pow BOOST_PREVENT_MACRO_SUBSTITUTION( STEPFAC3 , 1.0 / static_cast< value_type > ( 2*i+1 ) );
```

Comment:
Would just save little memory in comparison with the general properties of the BS dense stepper, I don't think saving the first component yields any benefit.